src/modules/hardwareWallet/store/reducers/reducers.js

```diff
@@ -0,0 +1,28 @@
+import actionTypes from '../actions/actionTypes';
+
+const initAccounts = [];
+const initState = {
+  deviceId: 0,
+  status: 'disconnected',
```

Comment:
please remove device id and status, we will have like below and they arent part of this ticket
store.hardwarewallet = { accounts, currentDevice, devices }

Reply:
@soroushm If I did that, it would affect line 12 (`const { deviceId, status } = useSelector(selectHW);`) in `src/modules/hardwareWallet/hooks/useManageHWAccounts.js` and its related tests